plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/PipelineStageViewHook.java

```diff
@@ -37,6 +43,13 @@ public static boolean isPipelineStageViewPlugin(Map<String, Object> moreInfo) {
     }
 
     public static boolean isPipelineStageViewPlugin(PomData data) {
-        return data.artifactId.contains("pipeline-rest-api");
+        if (data.parent != null) {
+            return data.parent.artifactId.equalsIgnoreCase("parent-pom");
```

Comment:
nit: maybe safer to check the groupId too here in case another plugin has a parent with the same artifactId.